.github/workflows/early_final_release_checker.yml

```
@@ -7,7 +7,7 @@ on:
       small_runner:
         description: 'Small runner: Comma-separated list of datasets to archive (e.g., "ferc2","ferc6").'
         # We can't pass env variables to the workflow_dispatch, so we manually list all small datasets here.
```

Not blocking but a tiny comment cleanup:

```suggestion
        # We can't pass env variables to the workflow_dispatch, so we manually list the target datasets here.
```